rac-gcp-deploy/executor/Dockerfile

```diff
@@ -2,27 +2,15 @@ ARG GCLOUD_SDK_VERSION
 FROM google/cloud-sdk:${GCLOUD_SDK_VERSION:-304.0.0-alpine}
 
 ARG SBT_VERSION
-ENV SBT_VERSION ${SBT_VERSION:-1.4.2}
+ENV SBT_VERSION ${SBT_VERSION:-1.4.7}
 
 RUN apk --update add openjdk8 jq
 
 RUN gcloud components install kubectl
```

Comment:
is kubectl still needed?

Reply:
Good spot. CircleCi does not interact with k8s so this can go away.